R/lmer_multimember.R

```diff
@@ -419,7 +419,7 @@ glmerModMultiMember <-
 #' @return summary of merModMultiMember object
 #' @export
 summary.merModMultiMember <- function(object, ...) {
-  if (!inherits(object, "merModMultiMember") && !inherits(object, "merMod")) {
```

Comment:
No point in allowing anything other than `merModMultiMember` class inheritance, because we use the multimembership weight matrix in the summary, so that really needs to be in the object we're summarizing.